README.md

```diff
@@ -207,6 +277,7 @@ Then drag the built `Katana.framework` into your Xcode project.
 
 - [Everyone at Bending Spoons](http://bendingspoons.com/team.html) for providing their priceless input;
```

Comment:
First line is ending with `;`, second line is ending with `.`, third line is ending with noting.